tests/console/nvme_checks.pm

```diff
@@ -77,6 +78,7 @@ sub _check_nvme_tools_installed {
         record_soft_failure "bsc#1172866 - nvme-cli is not installed";
         zypper_call("in nvme-cli");
     }
+    zypper_call("in pciutils") if (is_sle('16+'));
```

Comment:
why is needed that package? can you show the error?

Reply:
It is because of lspci cmd not found which belong to pciutils pkg. https://openqa.suse.de/tests/16816567#step/nvme_checks/7

Reply:
ok, that is a difference with sle 15, but we should not make it green, please check in #eng-testing about missing package for sles 16, please ensure first that that package is installed by default in sles or from which module was coming from.

Reply:
Ok, I think we can update the page https://openqa.suse.de/tests/16949058#details to add request to make the pkg installed by default on SLE16, and merge the PR with the pkg installed. Then if the request accepted, file a PR to remove the code to install the pkg. WDYT?

Reply:
we need to recheck after build that will have: https://github.com/agama-project/agama/pull/2116

Reply:
Yes, we don't need install it any more, https://openqa.suse.de/tests/16972140#step/nvme_checks/5

Reply:
updated code to do not install the pkg pciutils.